package.json

```diff
@@ -96,7 +96,8 @@
 		"isomorphic-form-data": "^1.0.0",
 		"node-fetch": "^2.0.1",
 		"tslib": "^1.13.0",
-		"ultra-runner": "^3.10.5"
+		"ultra-runner": "^3.10.5",
+		"dayjs": "^1.11.10"
```

Comment:
The [date-fns](https://github.com/date-fns/date-fns) package is currently a **devDependency** (only used [here](https://github.com/Esri/hub.js/blob/master/support/changelog.js#L6C29-L6C37) AFAIK), but the it doesn't support time zones out of the box; that requires also adding [date-fns-tz](https://github.com/marnusw/date-fns-tz). I tried bumping `date-fns` to latest but sadly the latest `date-fns-tz` is broken w/ the latest version of `date-fns` (also tested w/ latest `date-fns-tz: v3.0.0-beta.3`, published 4 days ago, to no avail). So adding `dayjs` as a dep to facilitate the adjustment of dates to the appropriate timezone so we post the right ISO date/time string to the server.